### ISSUE TITLE
Links to some GitHub pages & documentation prevents the file from being opened

### ISSUE THREAD

**Issue by TylerCarrol** [nathonius/obsidian-github-link#144]
I have documents with links to GitHub, such as the following:

- https://github.github.com/gfm/
- http://github.com/search
- https://docs.github.com/en/search-github
- https://docs.github.com/en/codespaces/overview

If I remove these links, the file can be opened.

If I remove everything *except* these links, the file cannot be opened, and I get the following error message:

```
TypeError: Failed to construct 'URL': Invalid URL
    at parseUrl2 (plugin:github-link:7173:15)
    at createTag (plugin:github-link:7787:21)
    at _InlineTagWidget.toDOM (plugin:github-link:7930:17)
    at t.sync (app.js:1:322514)
    at e.sync (app.js:1:313021)
    at t.sync (app.js:1:329284)
    at e.sync (app.js:1:313021)
    at app.js:1:354351
    at e.ignore (app.js:1:434969)
    at t.updateInner (app.js:1:354133)
    at new t (app.js:1:351885)
    at e.setState (app.js:1:444924)
    at t.set (app.js:1:1394173)
    at t.set (app.js:1:2332155)
    at t.setViewData (app.js:1:2347061)
    at t.setData (app.js:1:2025364)
    at t.<anonymous> (app.js:1:2025267)
    at app.js:1:237138
    at Object.next (app.js:1:237243)
    at a (app.js:1:235961)
```

**Comment by nathonius** [comment]
Sorry about that. It should at least be error tolerant, I'll fix that.

**Comment by nathonius** [comment]
@TylerCarrol can you actually give me an example file where you see this behavior? I can't replicate it. I'll definitely improve the handling but I'm not seeing that sort of issue with these links.

**Comment by nathonius** [comment]
Going to go ahead and close this since it should be fixed by the changes in #146, but please let me know if you can still reproduce this.

**Comment by TylerCarrol** [comment]
> Going to go ahead and close this since it should be fixed by the changes in #146, but please let me know if you can still reproduce this.

Seems to have worked, thanks for the quick fix!
